chugbot/index.php

```diff
@@ -56,15 +57,14 @@
 <h1><a>Welcome</a></h1>
 <form id="camperForm" class="form-group" method="GET">
 <div class="page-header">
-<h2>Welcome to the <?php echo $campName; ?> chug preference ranking system!</h3>
+<h2>Welcome to the <?php echo $campName; ?> <?php echo chug_term_singular ?> preference ranking system!</h3>
 <p><?php echo $codeMessage; ?></p>
 </div>
 <ul>
 
 <?php
 if (!$loggedIn) {
     $ccField = new FormItemSingleTextField("Camper Access Code", true, "camper_code", 0);
-    $ccField->setInputValue($adminEmail);
```

Comment:
This was pre-filling the field with the admin email address, which didn't seem like the desired UX to me